packages/firebase_ml_vision/ios/firebase_ml_vision.podspec

```diff
@@ -33,8 +33,8 @@ An SDK that brings Google's machine learning expertise to Android and iOS apps i
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'firebase_core'
-  s.dependency 'Firebase/MLVision', firebase_sdk_version
+  s.dependency 'Firebase/Core'
+  s.dependency 'Firebase/MLVision'
```

Comment:
Not sure why these changes were necessary? `firebase_core` is the FlutterFire plugin which transitively should bring in the Firebase/Core dependency? Additionally we use `firebase_sdk_version` to make sure all plugins use the same SDK version

Reply:
Reverting to see what CI does.